doc/source/user_guide/10min.rst

```diff
@@ -667,9 +667,10 @@ pandas can include categorical data in a :class:`DataFrame`. For full docs, see
 
 .. ipython:: python
 
-   df = pd.DataFrame(
-       {"id": [1, 2, 3, 4, 5, 6], "raw_grade": ["a", "b", "b", "a", "a", "e"]}
-   )
+    df = pd.DataFrame(
+        {"id": [1, 2, 3, 4, 5, 6], "raw_grade": ["a", "b", "b", "a", "a", "e"]}
+    )
```

Comment:
Looks like this block got extra-indented by one space, this is causing the CI error